test/integration/overload_integration_test.cc

```diff
@@ -849,4 +849,37 @@ TEST_P(LoadShedPointIntegrationTest, HttpConnectionMnagerCloseConnectionCreating
   EXPECT_EQ(response->headers().getStatusValue(), "200");
 }
 
+TEST_P(LoadShedPointIntegrationTest, HttpDownstreamFilterLoadShed) {
+  if (downstreamProtocol() == Http::CodecClient::Type::HTTP3) {
```

Comment:
what's the reason we skip this for QUIC? would expect it to also work there?

Reply:
Good catch, it should work for h/3, done